recipe/meta.yaml

```diff
@@ -1,113 +1,86 @@
-    test:
-      requires:
-        - pip
-      commands:
-        - pip check
-    about:
-      home: https://github.com/fchollet/keras
-      license: MIT
-      license_family: MIT
-      license_file: LICENSE
-      summary: Deep Learning Library for Theano and TensorFlow
-      description: |
-        Keras is a minimalist, highly modular neural networks library
-        written in Python and capable on running on top of either
-        TensorFlow or Theano. It was developed with a focus on enabling
-        fast experimentation. Being able to go from idea to result with
-        the least possible delay is key to doing good research.
-      doc_url: http://keras.io
-      dev_url: https://github.com/fchollet/keras
+requirements:
+  build:
+    - patch    #  [not win]
+    - m2-patch #  [win]
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    - absl-py
+    - numpy
+    - rich
+    - namex
+    - h5py
+    - ml_dtypes
+    - packaging
+    - dm-tree
 
-  - name: keras-gpu
-    build:
-      noarch: true
-    requirements:
-      run:
-        - keras-base {{ version }}
-        - tensorflow-gpu
-    about:
-      home: https://github.com/fchollet/keras
-      license: MIT
-      license_family: MIT
-      license_file: LICENSE
-      summary: Deep Learning Library for Theano and TensorFlow
-      description: |
-        Keras is a minimalist, highly modular neural networks library
-        written in Python and capable on running on top of either
-        TensorFlow or Theano. It was developed with a focus on enabling
-        fast experimentation. Being able to go from idea to result with
-        the least possible delay is key to doing good research.
-      doc_url: http://keras.io
-      dev_url: https://github.com/fchollet/keras
+  run_constrained:
+    - tensorflow >=2.15.0,<2.17.0a
+    - jax 0.4.*
+    - pytorch >=2.1,<2.3
+
+test:
+  imports:
+    - keras
+    - keras.backend
+    - keras.datasets
+    - keras.layers
+    - keras.utils
+  requires:
+    - pip
+    - jax  # [not win]
+    - pytorch
+  commands:
+    - pip check
+    # numpy import jax in source code which causes test to fail for win
+    - python -c "import os; os.environ['KERAS_BACKEND'] = 'numpy'; import keras;"  # [not win]
+    - python -c "import os; os.environ['KERAS_BACKEND'] = 'jax'; import keras;"  # [not win]
+    - python -c "import os; os.environ['KERAS_BACKEND'] = 'torch'; import keras;"
+
+about:
+  home: https://github.com/fchollet/keras
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Deep Learning Library for Theano and TensorFlow
+  description: |
+    Keras is a minimalist, highly modular neural networks library
+    written in Python and capable on running on top of either
+    TensorFlow or Theano. It was developed with a focus on enabling
+    fast experimentation. Being able to go from idea to result with
+    the least possible delay is key to doing good research.
```

Comment:
This needs an update, as `Theano` is no longer supported.

Reply:
Updated the summary and description.